internal/dao/cronjob.go

```diff
@@ -17,7 +17,7 @@ import (
 
 const (
 	maxJobNameSize = 42
-	cronJobGVR     = "batch/v1beta1/cronjobs"
+	cronJobGVR     = "batch/v1/cronjobs"
```

Comment:
If someone uses the old v1beta1 version we will show an error in this case? Or maybe someone has both versions in his cluster.  Is it possible to determine the right cj version in this case and use it?